pkg/lsp/service.go

```diff
@@ -254,7 +255,7 @@ func (s *ServiceImpl) NotifyDidClose(ctx context.Context, file model.File) error
 
 	if !ok {
 		log.Error().Msg("Project not found in context")
-		return fmt.Errorf("Project not found in context")
+		return errors.New("project not found in context")
```

Comment:
BTW errors should not be capitalised. We can nit this at some point.

Reply:
TIL